.sync.yml

```diff
@@ -0,0 +1,54 @@
+---
```

Comment:
@mcgege something new for me:) What is the purpose of this file? It looks like some tool, which I do not know yet :)